lua/entities/gmod_wire_coldetector.lua

```diff
@@ -0,0 +1,224 @@
+AddCSLuaFile()
+
+DEFINE_BASECLASS( "base_wire_entity" )
+ENT.PrintName       = "Wire Collision Detector"
+ENT.WireDebugName	= "Collision Detector"
+
+if CLIENT then return end
+
+local stickplayer=CreateConVar( "wire_col_detector_stickplayer", 0, 0, "Allow linked entity stick to players" )
+local stickprops=CreateConVar( "wire_col_detector_stickprops", 0, 0, "Allow sticking to other players stuff" )
+
+function entitiesCollide(ent,data)
+	if !IsValid(ent.ColSensor) or ent.ColSensor.LinkedEnt != ent then return end
+	ent.ColSensor:ApplyCollide(data,ent)
+end
+
+function ENT:Initialize()
+	self:PhysicsInit( SOLID_VPHYSICS )
```

Comment:
Personally I do use spaces inside parantheses, but in my opinion it does not really affect readability and it's up to you to choose what to use. Wiremod shouldn't be so strict imo